### ISSUE TITLE
Upgrade to Vuetify 2.x when released

### ISSUE THREAD

**Issue by manastalukdar** [manastalukdar/manastalukdar.github.io#48]
References:
- https://github.com/vuetifyjs/vuetify/issues/6676

**Comment by manastalukdar** [comment]
Issues:
- https://github.com/vuetifyjs/vuetify/issues/7879
- https://github.com/vuetifyjs/vuetify/issues/7410

**Comment by manastalukdar** [comment]
Repro gif for 7879:

![vuetify2_7879_repro](https://user-images.githubusercontent.com/2075466/61583571-e3112200-aaee-11e9-8495-074e410a496e.gif)

**Comment by manastalukdar** [comment]
[{"published": true, "tags": [{"name": "Algorithms", "url-slug": "algorithms"}, {"name": "Computer Science", "url-slug": "computer-science"}, {"name": "Sliding Window", "url-slug": "sliding-window"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Sliding Window Algorithm", "url-slug": "sliding-window-algorithm", "first-published-on": "2019-07-12T13:45:00", "last-updated-on": "2019-07-12T13:45:00", "meta": {"description": "An exposition of the sliding window algorithm."}, "excerpt": "The best way to understand the sliding window algorithm is to work with an example. Say we have an array of integers,", "path": "2019/07/12/sliding-window-algorithm/readme.md"}, {"published": true, "tags": [{"name": "Conference Talk", "url-slug": "conference-talk"}, {"name": "DataWorks Summit", "url-slug": "dataworks-summit"}, {"name": "Distributed Systems", "url-slug": "distributed-systems"}, {"name": "Scaling", "url-slug": "scaling"}, {"name": "Uber", "url-slug": "uber"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "video", "url-slug": "video"}, "title": "How Uber Scaled its Real Time Infrastructure to Trillion Events per Day", "url-slug": "uber-scaling-real-time-infrastructure", "first-published-on": "2019-06-26T00:45:00", "last-updated-on": "2019-06-26T14:28:00", "meta": {"description": "Conference talk on Uber scaling its real-time infrastructure."}, "excerpt": "Conference talk on Uber scaling its real-time infrastructure.", "path": "2019/06/26/uber-scaling-real-time-infrastructure/readme.md"}, {"published": true, "tags": [{"name": "Language", "url-slug": "language"}, {"name": "Learning", "url-slug": "learning"}, {"name": "Russian", "url-slug": "russian"}], "categories": [{"name": "Miscellaneous", "url-slug": "miscellaneous"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "aside", "url-slug": "aside"}, "title": "Learning Russian", "url-slug": "learning-russian", "first-published-on": "2019-06-21T11:00:00", "last-updated-on": "2019-06-21T18:08:00", "meta": {"description": "Blog post on starting to learn Russian and the resources being used for this project."}, "excerpt": "I have started learning Russian recently. Growing up I read a lot of Russian literature (English translations).", "path": "2019/06/21/learning-russian/readme.md"}, {"published": true, "tags": [{"name": "Cloud Computing", "url-slug": "cloud-computing"}, {"name": "Distributed Systems", "url-slug": "distributed-systems"}, {"name": "Service Operations", "url-slug": "service-operations"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "link", "url-slug": "link"}, "title": "GitLab Runbooks", "url-slug": "gitlab-runbooks", "first-published-on": "2019-05-28T14:48:00", "last-updated-on": "2019-05-28T14:48:00", "meta": {"description": "Link to GitLab runbooks."}, "excerpt": "Link to GitLab runbooks", "path": "2019/05/28/gitlab-runbooks/readme.md"}, {"published": true, "tags": [{"name": "Book", "url-slug": "book"}, {"name": "Cloud Computing", "url-slug": "cloud-computing"}, {"name": "Distributed Systems", "url-slug": "distributed-systems"}, {"name": "Microservices", "url-slug": "microservices"}, {"name": "Reading", "url-slug": "reading"}, {"name": "Service Operations", "url-slug": "service-operations"}, {"name": "Software Engineering", "url-slug": "software-engineering"}, {"name": "Technical Literature", "url-slug": "technical-literature"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "aside", "url-slug": "aside"}, "title": "Distributed Systems Book - Production-Ready Microservices", "url-slug": "distributed-systems-book-production-ready-microservices", "first-published-on": "2019-05-28T14:25:00", "last-updated-on": "2019-05-28T14:48:00", "meta": {"description": "Note on a book on running distributed systems in production."}, "excerpt": "I recently read the book, Production-Ready Microservices. This book has some good pointers on what it takes to run \"operations for live services\".", "path": "2019/05/28/distributed-systems-book-production-ready-microservices/readme.md"}, {"published": true, "tags": [{"name": "Agile Methodology", "url-slug": "agile-methodology"}, {"name": "Engineering Leadership", "url-slug": "engineering-leadership"}, {"name": "Project Management", "url-slug": "project-management"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Measuring Productivity and Performance of Software Engineering Teams", "url-slug": "measuring-productivity-performance-software-engineering-teams", "first-published-on": "2019-05-04T13:15:00", "last-updated-on": "2019-05-09T18:25:00", "meta": {"description": "A data-driven methodology for measuring productivity and performance of software engineering teams."}, "excerpt": "A key facet of running software engineering teams is being able to measure team performance over time.", "path": "2019/05/04/measuring-productivity-performance-software-engineering-teams/readme.md"}, {"published": true, "tags": [{"name": "Engineering Leadership", "url-slug": "engineering-leadership"}, {"name": "Project Management", "url-slug": "project-management"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Area Champions in Software Engineering Teams", "url-slug": "area-champions-software-engineering-teams", "first-published-on": "2019-05-01T12:56:00", "last-updated-on": "2019-05-01T13:10:00", "meta": {"description": "A methodology for keeping software engineers interested and engaged, preventing compartmentalization and spreading knowledge across the team."}, "excerpt": "An important goal in running software engineering teams is ensuring software engineers remain interested and engaged in their work.", "path": "2019/05/01/area-champions-software-engineering-teams/readme.md"}, {"published": true, "tags": [{"name": "Debugging", "url-slug": "debugging"}, {"name": "Memory Dumps", "url-slug": "memory-dumps"}, {"name": "WinDbg", "url-slug": "windbg"}, {"name": "Windows", "url-slug": "windows"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Working with Memory Dumps on Windows using WinDbg", "url-slug": "working-with-memory-dumps-windows-windbg", "first-published-on": "2019-04-29T10:55:00", "last-updated-on": "2019-04-29T10:55:00", "meta": {"description": "Some steps on getting and analyzing memory dumps on Windows using WinDbg."}, "excerpt": "Open WinDbg and go to File > Attach to a Process. Select the process from the list and hit OK.", "path": "2019/04/29/working-with-memory-dumps-windows-windbg/readme.md"}, {"published": true, "tags": [{"name": "Git", "url-slug": "git"}, {"name": "Gitflow", "url-slug": "gitflow"}, {"name": "Source Control", "url-slug": "source-control"}, {"name": "Team Foundation Version Control", "url-slug": "team-foundation-version-control"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Git vs. Team Foundation Version Control", "url-slug": "git-vs-team-foundation-version-control", "first-published-on": "2019-04-29T10:50:00", "last-updated-on": "2019-05-02T12:47:00", "meta": {"description": "A comparison of Git vs. Team Foundation Version Control with motivations for moving to Git."}, "excerpt": "The primary motivation for moving to Git is the far better support for branching and merging.", "path": "2019/04/29/git-vs-team-foundation-version-control/readme.md"}, {"published": true, "tags": [{"name": "Agile Methodology", "url-slug": "agile-methodology"}, {"name": "Continuous Integration", "url-slug": "continuous-integration"}, {"name": "Git", "url-slug": "git"}, {"name": "Gitflow", "url-slug": "gitflow"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Continuous Integration using Git Branching and Pull Requests", "url-slug": "continuous-integration-git-branching-pull-requests", "first-published-on": "2019-04-28T16:15:00", "last-updated-on": "2019-04-28T16:15:00", "meta": {"description": "An implementation of continuous integration using Git branching and pull requests."}, "excerpt": "I like to follow the Gitflow branching model with slight variations.", "path": "2019/04/28/continuous-integration-git-branching-pull-requests/readme.md"}, {"published": true, "tags": [{"name": "Agile Methodology", "url-slug": "agile-methodology"}, {"name": "Engineering Leadership", "url-slug": "engineering-leadership"}, {"name": "Project Management", "url-slug": "project-management"}, {"name": "Scrum", "url-slug": "scrum"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Development Methodology", "url-slug": "development-methodology", "first-published-on": "2019-04-28T13:15:00", "last-updated-on": "2019-04-30T18:36:00", "meta": {"description": "An exposition of the software development methodology in engineering teams I run."}, "excerpt": "Software Engineering teams I run follow agile development process, specifically scrum, or a close variation of it.", "path": "2019/04/28/development-methodology/readme.md"}, {"published": true, "tags": [{"name": "Engineering Leadership", "url-slug": "engineering-leadership"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Attributes of a Senior Level Software Engineer", "url-slug": "attributes-senior-level-software-engineer", "first-published-on": "2019-04-17T11:30:00", "last-updated-on": "2019-04-17T11:30:00", "meta": {"description": "An exposition of ideal attributes expected to be exhibited by a senior level software engineer."}, "excerpt": "A senior software engineer is expected to be highly technically competent from both coding and system design perspective.", "path": "2019/04/17/attributes-senior-level-software-engineer/readme.md"}, {"published": true, "tags": [{"name": "Cloud Computing", "url-slug": "cloud-computing"}, {"name": "Computer Science", "url-slug": "computer-science"}, {"name": "Course", "url-slug": "course"}, {"name": "Data Management", "url-slug": "data-management"}, {"name": "Distributed Systems", "url-slug": "distributed-systems"}, {"name": "Learning", "url-slug": "learning"}, {"name": "Software Engineering", "url-slug": "software-engineering"}, {"name": "Stanford", "url-slug": "stanford"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "link", "url-slug": "link"}, "title": "Stanford CS Course on Parallel and Distributed Data Management", "url-slug": "parallel-distributed-data-management-stanford-cs-course", "first-published-on": "2019-04-15T10:45:00", "last-updated-on": "2019-04-15T10:45:00", "meta": {"description": "Link to Stanford CS Course on Parallel and Distributed Data Management."}, "excerpt": "Link to Stanford CS Course on Parallel and Distributed Data Management.", "path": "2019/04/15/parallel-distributed-data-management-stanford-cs-course/readme.md"}, {"published": true, "tags": [{"name": "Engineering Leadership", "url-slug": "engineering-leadership"}, {"name": "Hiring", "url-slug": "hiring"}, {"name": "Interviewing", "url-slug": "interviewing"}, {"name": "Recruiting", "url-slug": "recruiting"}, {"name": "Software Engineer Interview", "url-slug": "software-engineer-interview"}, {"name": "Software Engineering", "url-slug": "software-engineering"}, {"name": "Technical Interviews", "url-slug": "technical-interviews"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Interviewing Software Engineers", "url-slug": "interviewing-software-engineers", "first-published-on": "2019-04-14T13:20:00", "last-updated-on": "2019-04-15T17:30:00", "meta": {"description": "Methodology for interviewing and hiring high calibre software engineers."}, "excerpt": "I have been interviewing software engineer candidates, both new grads and experienced candidates, for many years now. Over the years I have settled down on a pattern of questions that have resulted in hiring excellent software engineers with a high degree of probability.", "path": "2019/04/14/interviewing-software-engineers/readme.md"}, {"published": true, "tags": [{"name": "Hiring", "url-slug": "hiring"}, {"name": "Interviewing", "url-slug": "interviewing"}, {"name": "Joel Spolsky", "url-slug": "joel-spolsky"}, {"name": "Recruiting", "url-slug": "recruiting"}, {"name": "Software Engineer Interview", "url-slug": "software-engineer-interview"}, {"name": "Software Engineering", "url-slug": "software-engineering"}, {"name": "Technical Interviews", "url-slug": "technical-interviews"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "link", "url-slug": "link"}, "title": "The Guerrilla Guide to Interviewing (version 3.0) by Joel Spolsky", "url-slug": "guerrilla-guide-interviewing-v3-joel-spolsky", "first-published-on": "2019-04-14T00:15:00", "last-updated-on": "2019-04-14T00:15:00", "meta": {"description": "The Guerrilla Guide to Interviewing version 3.0 by Joel Spolsky."}, "excerpt": "The Guerrilla Guide to Interviewing (version 3.0) by Joel Spolsky.", "path": "2019/04/14/guerrilla-guide-interviewing-v3-joel-spolsky/readme.md"}, {"published": true, "tags": [{"name": "Behavioral Psychology", "url-slug": "behavioral-psychology"}, {"name": "Conference", "url-slug": "conference"}, {"name": "Evolutionary Psychology", "url-slug": "evolutionary-psychology"}, {"name": "IIot", "url-slug": "iiot"}, {"name": "OSIsoft", "url-slug": "osisoft"}, {"name": "OSIsoft Cloud Services", "url-slug": "osisoft-cloud-services"}, {"name": "OSIsoft Edge Data Store", "url-slug": "osisoft-edge-data-store"}, {"name": "PI World", "url-slug": "pi-world"}, {"name": "PI World 2019", "url-slug": "pi-world-2019"}, {"name": "Process Data", "url-slug": "process-data"}, {"name": "Psychology", "url-slug": "psychology"}, {"name": "San Francisco", "url-slug": "san-francisco"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Brief Summary of PI World 2019", "url-slug": "pi-world-2019-san-francisco-brief-summary", "first-published-on": "2019-04-13T00:00:00", "last-updated-on": "2019-04-13T00:00:00", "meta": {"description": "PI World 2019 in San Francisco had a certain focus on cloud and edge. Customer were very interested in solutions for both domains and the keynote demo helped drive some of that interest."}, "excerpt": "I was at PI World 2019 in San Francisco last week. It was hectic, but also quite interesting seeing the industrial software domain veer towards cloud and edge.", "path": "2019/04/13/pi-world-2019-san-francisco-brief-summary/readme.md"}, {"published": true, "tags": [{"name": "Book", "url-slug": "book"}, {"name": "Cloud Computing", "url-slug": "cloud-computing"}, {"name": "Distributed Systems", "url-slug": "distributed-systems"}, {"name": "eBook", "url-slug": "ebook"}, {"name": "Reading", "url-slug": "reading"}, {"name": "Software Engineering", "url-slug": "software-engineering"}, {"name": "Technical Literature", "url-slug": "technical-literature"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "aside", "url-slug": "aside"}, "title": "eBook on Distributed Systems - Distributed systems for fun and profit", "url-slug": "distributed-systems-fun-profit-ebook", "first-published-on": "2019-04-08T00:00:00", "last-updated-on": "2019-04-08T00:00:00", "meta": {"description": "Note on an eBook for Distributed Systems."}, "excerpt": "I recently read this (free and open source) e-book on distributed systems.", "path": "2019/04/08/distributed-systems-fun-profit-ebook/readme.md"}, {"published": true, "tags": [{"name": "Cloud Computing", "url-slug": "cloud-computing"}, {"name": "Conference Paper", "url-slug": "conference-paper"}, {"name": "Distributed Systems", "url-slug": "distributed-systems"}, {"name": "Microservices", "url-slug": "microservices"}, {"name": "Microsoft Azure", "url-slug": "microsoft-azure"}, {"name": "Service Fabric", "url-slug": "service-fabric"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "link", "url-slug": "link"}, "title": "Service Fabric Paper from EuroSys 2018", "url-slug": "service-fabric-paper-eurosys-2018", "first-published-on": "2019-04-07T00:00:00", "last-updated-on": "2019-04-07T00:00:00", "meta": {"description": "Service Fabric Paper from EuroSys 2018"}, "excerpt": "Service Fabric Paper from EuroSys 2018", "path": "2019/04/07/service-fabric-paper-eurosys-2018/readme.md"}, {"published": true, "tags": [{"name": "Ashoka", "url-slug": "ashoka"}, {"name": "Greeks", "url-slug": "greeks"}, {"name": "India", "url-slug": "india"}, {"name": "Indian History", "url-slug": "indian-history"}, {"name": "Maurya", "url-slug": "maurya"}, {"name": "Ramesh Chandra Majumdar", "url-slug": "ramesh-chandra-majumdar"}], "categories": [{"name": "History", "url-slug": "history"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "aside", "url-slug": "aside"}, "title": "RC Majumdar on Ashoka and the downfall of the Mauryan empire", "url-slug": "rc-majumdar-ashoka-downfall-mauryan-empire", "first-published-on": "2019-04-05T00:00:00", "last-updated-on": "2019-04-05T00:00:00", "meta": {"description": "RC Majumdar on Ashoka and the downfall of the Mauryan empire."}, "excerpt": "RC Majumdar makes no bones about the root cause behind the downfall of the mighty Mauryan Empire. The cause of course being the overt renunciation of military might by Ashoka. Let the master himself speak", "path": "2019/04/05/rc-majumdar-ashoka-downfall-mauryan-empire/readme.md"}, {"published": true, "tags": [{"name": "Cloud Computing", "url-slug": "cloud-computing"}, {"name": "Conference Talk", "url-slug": "conference-talk"}, {"name": "Cosmos DB", "url-slug": "cosmos-db"}, {"name": "Data Consistency", "url-slug": "data-consistency"}, {"name": "Distributed Systems", "url-slug": "distributed-systems"}, {"name": "Facebook @Scale Conference", "url-slug": "facebook-scale-conference"}, {"name": "Leslie Lamport", "url-slug": "leslie-lamport"}, {"name": "Microsoft Azure", "url-slug": "microsoft-azure"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "video", "url-slug": "video"}, "title": "Azure Cosmos DB and Consistency Levels", "url-slug": "azure-cosmos-db-consistency-levels", "first-published-on": "2019-04-04T00:00:00", "last-updated-on": "2019-04-04T00:00:00", "meta": {"description": "Azure Cosmos DB talk mentioning the different consistency levels used in the globally distributed database service."}, "excerpt": "Azure Cosmos DB talk mentioning the different consistency levels used in the globally distributed database service.", "path": "2019/04/04/azure-cosmos-db-consistency-levels/readme.md"}, {"published": true, "tags": [{"name": "Adaptive Concurrency", "url-slug": "adaptive-concurrency"}, {"name": "Availability", "url-slug": "availability"}, {"name": "Cloud Computing", "url-slug": "cloud-computing"}, {"name": "Concurrency", "url-slug": "concurrency"}, {"name": "Distributed Systems", "url-slug": "distributed-systems"}, {"name": "Load", "url-slug": "load"}, {"name": "Netflix", "url-slug": "netflix"}, {"name": "Netflix TechBlog", "url-slug": "netflix-techblog"}, {"name": "Performance", "url-slug": "performance"}, {"name": "Software Engineering", "url-slug": "software-engineering"}], "categories": [{"name": "Technology", "url-slug": "technology"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "link", "url-slug": "link"}, "title": "Performance Under Load - Adaptive Concurrency Limits at Netflix", "url-slug": "performance-under-load-adaptive-concurrency-limits-netflix", "first-published-on": "2019-04-04T00:00:00", "last-updated-on": "2019-04-04T00:00:00", "meta": {"description": "Adaptive concurrency limits at Netflix for performance under load."}, "excerpt": "Adaptive concurrency limits at Netflix for performance under load.", "path": "2019/04/04/performance-under-load-adaptive-concurrency-limits-netflix/readme.md"}, {"published": true, "tags": [{"name": "Blog", "url-slug": "blog"}, {"name": "Nuxt.js", "url-slug": "nuxt-js"}, {"name": "Personal Website", "url-slug": "personal-website"}, {"name": "Vue.js", "url-slug": "vue-js"}], "categories": [{"name": "Miscellaneous", "url-slug": "miscellaneous"}], "authors": [{"name": "Manas Talukdar", "url-slug": "manas-talukdar"}], "post-format": {"name": "standard", "url-slug": "standard"}, "title": "Getting Started", "url-slug": "getting-started", "first-published-on": "2019-03-28T00:00:00", "last-updated-on": "2019-06-21T00:00:00", "meta": {"title": "Getting Started", "description": "Getting started post."}, "excerpt": "Welcome to my blog. I recently built this website (with blog). I learnt Vue.js and Nuxt.js for this project and it was a lot of fun.", "path": "2019/03/28/getting-started/readme.md"}]
